### ISSUE TITLE
Add more benchmark apps

### ISSUE THREAD

**Issue by markerikson** [reduxjs/react-redux-benchmarks#7]
We can start adding more benchmark apps now.

Pasting some other benchmark-ish repos that are worth investigating and potentially porting:

https://github.com/mweststrate/redux-todomvc
https://github.com/jscriptcoder/stressing-redux
https://github.com/somebody32/high-performance-redux/tree/gh-pages/assets
https://github.com/coderitual/redux-performance
https://github.com/reduxjs/react-redux/issues/686#issuecomment-331261897
https://github.com/Kalkut/redux-data-frequency-benchmark

Also, we should look at writing some more "real-world" benchmarks, not just FPS stress tests, but that may depend on figuring out what other metrics we want to measure (#3 ).

**Comment by markerikson** [comment]
Another potential source: the JS framework benchmarks repo at https://github.com/krausest/js-framework-benchmark .

**Comment by Avi98** [comment]
@markerikson  i can start implementing https://github.com/mweststrate/redux-todomvc and refer stockticker benchmark

**Comment by markerikson** [comment]
@Avi98: sure, go ahead, thanks!

**Comment by Avi98** [comment]
Hi @markerikson,  i need your help in providing some direction for setting up "react-redux-benchmarks" . I followed steps provided in the README.md.  but while running  this command  " npm run initialize" i'm getting the following error. please help me. Thank you
<img width="1438" alt="screen shot 2018-10-04 at 11 33 50 pm" src="https://user-images.githubusercontent.com/26133749/46568127-7b258c80-c95d-11e8-9711-03ced1c62f6e.png">

**Comment by markerikson** [comment]
Hmm.  Not exactly sure what's going on there.  Looks like NPM was having trouble copying a file during the install process.  Does this happen every time?

**Comment by Avi98** [comment]
Yes, steps i followed for setting up the project after cloning it from github 
`cd react-redux-benchmarks`
 `npm install`
`npm run initialize` 
and its still showing the error i have deleted node modules reinstalled it but its no help. Any alternative.
Thank you

**Comment by dai-shi** [comment]
@Avi98 I have the same error and end up with a patch. I'm not sure what is a problem. Hope it's somehow fixed.

[diff.txt](https://github.com/reduxjs/react-redux-benchmarks/files/2706540/diff.txt)

```bash
git clone https://github.com/reduxjs/react-redux-benchmarks.git
cd react-redux-benchmarks
patch -p1 < ../diff.txt
npm install
npm run initialize
npm start
```

**Comment by dai-shi** [comment]
Hm, it doesn't seem to work as expected. Sorry.

**Comment by dai-shi** [comment]
It works now.

[diff2.txt](https://github.com/reduxjs/react-redux-benchmarks/files/2707457/diff2.txt)

```
git clone https://github.com/reduxjs/react-redux-benchmarks.git
cd react-redux-benchmarks
patch -p1 < ../diff2.txt
curl https://unpkg.com/react@16.6.0/umd/react.production.min.js > copy-to-public/react.production.min.js
npm install
npm run initialize
BENCHMARKS=twitter-lite npm start -s
Running benchmark twitter-lite
  react-redux version: 5.0.7
    Checking max FPS... (30 seconds)
    Running trace...    (30 seconds)
  react-redux version: 6.0-1000-4855c84
    Checking max FPS... (30 seconds)
    Running trace...    (30 seconds)

Results for benchmark twitter-lite:
┌──────────────────┬─────────┬───────────┬───────────┬──────────┬───────────────────────────────────────────────────────────────────────────────┐
│ Version          │ Avg FPS │ Scripting │ Rendering │ Painting │ FPS Values                                                                    │
├──────────────────┼─────────┼───────────┼───────────┼──────────┼───────────────────────────────────────────────────────────────────────────────┤
│ 5.0.7            │ 45.73   │ 20656.72  │ 3581.47   │ 373.67   │ 54,59,60,57,59,52,59,57,54,45,57,55,51,43,37,42,37,36,32,30,29,28,27          │
├──────────────────┼─────────┼───────────┼───────────┼──────────┼───────────────────────────────────────────────────────────────────────────────┤
│ 6.0-1000-4855c84 │ 31.76   │ 24595.08  │ 2232.00   │ 284.90   │ 49,60,55,59,47,58,49,48,39,35,32,30,29,25,22,24,23,22,21,16,18,17,18,17,16,14 │
└──────────────────┴─────────┴───────────┴───────────┴──────────┴───────────────────────────────────────────────────────────────────────────────┘
```

I'll file a new issue about the broken file.

**Comment by Avi98** [comment]
hi @dai-shi, I don't remember but I think I removed all lock file and reinstalled then it started working.
What did you do?

**Comment by dai-shi** [comment]
@Avi98 Is it that easy? I'll try it later.

Anyway, since I already posted a bunch of unrelated comments and you have no problem now,
I'll stop this topic here. If we need further discussions, let's open a new issue.

**Comment by markerikson** [comment]
More relevant examples we can draw from:

https://medium.com/@ryansolid/react-hooks-the-most-performant-way-to-write-react-393e135e1cc
